### PR TITLE
log style tweak, add log level option,  show process name and path in connections page

### PR DESCRIPTION
**Pull request by MetaCubeX**
Chore: log style tweak
Feat: add log level option, show process name and path in connections page

### CODE REVIEW

**Comment by MetaCubeX** [comment]
Screenshot 2022-04-15 062119
Screenshot 2022-04-15 062149
Screenshot 2022-04-15 062221

File: src/containers/Connections/index.tsx [hunk 1]
```diff
@@ -99,6 +101,7 @@ export default function Connections () {
     const columns = useMemo(
         () => table.createColumns([
             table.createDataColumn(Columns.Host, { minWidth: 260, width: 260, header: t(`columns.${Columns.Host}`) }),
+            table.createDataColumn(Columns.Process, { minWidth: 100, width: 130, header: t(`columns.${Columns.Process}`) }),
```
Comment (Dreamacro) [inline src/containers/Connections/index.tsx:104]: `process` has a value only if there is a relevant rule, it should not put it in front

File: src/lib/request.ts [hunk 2]
```diff
@@ -80,6 +80,8 @@ export interface Connections {
         network: string
         type: string
         host: string
+        process: string
```
Comment (Dreamacro) [inline src/lib/request.ts:83]: there is no process field

File: src/lib/request.ts [hunk 3]
```diff
@@ -80,6 +80,8 @@ export interface Connections {
...
+        processPath: string
```
Comment (Dreamacro) [inline src/lib/request.ts:84]: shoule be `processPath?: string`

File: src/i18n/zh_CN.ts [hunk 4]
```diff
@@ -16,6 +16,8 @@ const CN = {
             setAsSystemProxy: '设置为系统代理',
             allowConnectFromLan: '允许来自局域网的连接',
             proxyMode: '代理模式',
+            logLevel: '信息等级',
```
Comment (Dreamacro) [inline src/i18n/zh_CN.ts:19]: should be 日志等级

File: src/containers/Settings/index.tsx [hunk 5]
```diff
@@ -95,89 +107,111 @@ export default function Settings () {
...
-                </div>
-                <div className="flex flex-wrap">
-                    <div className="flex w-full py-3 px-8 items-center justify-between md:w-1/2">
+                    <div className="flex flex-wrap w-full py-3 px-8 items-center justify-between md:w-1/2">
                         <span className="font-bold label">{t('labels.externalController')}</span>
                         <span
-                            className={classnames({ 'modify-btn': !isClashX }, 'external-controller')}
-                            onClick={() => !isClashX && setIdentity(false)}>
+                            className={classnames('external-controller')}
+                            onClick={() => setIdentity(false)}>
```
Comment (Dreamacro) [inline src/containers/Settings/index.tsx:211]: clashx related condition should not delete

File: src/containers/Settings/index.tsx [hunk 6]
```diff
@@ -95,89 +107,111 @@ export default function Settings () {
...
+        { label: ('info'), value: 'info' },
+        { label: ('warn'), value: 'warning' },
+        { label: ('error'), value: 'error' },
+        { label: ('silent'), value: 'silent' },
+    ]
+
     return (
         <div className="page">
             <Header title={t('title')} />
+            <>{
```
Comment (Dreamacro) [inline src/containers/Settings/index.tsx:121]: The fragment here is meaningless

File: src/containers/Settings/index.tsx [hunk 7]
```diff
@@ -95,89 +107,111 @@ export default function Settings () {
         return options
     }, [t, premium])
 
+    const logLevelOptions = [
+        { label: ('debug'), value: 'debug' },
```
Comment (Dreamacro) [inline src/containers/Settings/index.tsx:111]: parenthesis is meaningless, and the first letter should be capitalized

File: src/containers/Logs/index.tsx [hunk 8]
```diff
@@ -15,7 +16,12 @@ export default function Logs () {
     const { t } = translation('Logs')
     const logsStreamReader = useLogsStreamReader()
     const scrollHeightRef = useRef(listRef.current?.scrollHeight ?? 0)
-
+    const InfoColors = {
```
Comment (Dreamacro) [inline src/containers/Logs/index.tsx:19]: It seems that this is not used

File: src/containers/Logs/index.tsx [hunk 9]
```diff
@@ -47,9 +53,17 @@ export default function Logs () {
                     {
                         logs.map(
                             (log, index) => (
-                                <li className="leading-5 inline-block" key={index}>
-                                    <span className="mr-4 text-gray-400 text-opacity-90">{ dayjs(log.time).format('YYYY-MM-DD HH:mm:ss') }</span>
-                                    <span>[{ log.type }] { log.payload }</span>
+                                <li className="leading-5 inline-block " style={{ fontSize: '11px' }} key={index}>
```
Comment (Dreamacro) [inline src/containers/Logs/index.tsx:56]: should not use style object, use windicss classname instead

**Review by Dreamacro — CHANGES_REQUESTED** [review]
All clashx related conditions should not delete

File: src/containers/Connections/index.tsx (hunk 1 above)
Comment (MetaCubeX) [reply src/containers/Connections/index.tsx:104]: Is `Process` moved to the back display? 
But that doesn't make it easy to look up the rules. 
How about showing `empty` when Process is empty?

Comment (Dreamacro) [reply src/containers/Connections/index.tsx:104]: maybe you can put it after rule

**Comment by MetaCubeX** [comment]
没搓完，不小心点提交了，改好再提交哈

**Comment by Dreamacro** [comment]
就在这个 pr commit 吧，不要再开新的 pr 了

**Comment by MetaCubeX** [comment]
> 就在这个 pr commit 吧，不要再开新的 pr 了

好的老大，我顺手把适配苹果的启动页搓进去可以吗……

**Pull request closed without merging**